runtimes/python/object-read.py

```diff
@@ -18,11 +18,22 @@ def main(args):
   resultsGetParams = getParamsCOS(args)
   cos = resultsGetParams['cos']
   params = resultsGetParams['params']
-  object = cos.get_object(
-    Bucket=params['Bucket'],
-    Key=params['Key'],
+  bucket = params['Bucket']
+  key = params['Key']
+  try:
+    object = cos.get_object(
+    Bucket=bucket,
+    Key=key,
   )
-  return {'data': str(object['Body'].read()).encode(encoding='UTF-8')}
+  except ClientError as e:
+    print(e)
+    raise e
+
+  return {
+  'Bucket':bucket,
+  'Key':key,
+  'Data': str(object['Body'].read()).encode(encoding='UTF-8')
```

Comment:
Do all other properties read from COS here and in other actions need to be in UTF-8 as well?

Reply:
need to do a little more research into what the encoding we return should be.  I think @csantanapr just used UTF-8 as a starting point.  I'll add an issue to this repo.

Reply:
Updated this to just directly return the body -- I believe this will always (?) be a buffer that gets returned, and the user can then decode how they want.  Will still open an issue & do more looking into it.